packages/react-ui/src/components/dialog/index.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { Element } from '../../primitives'
+import * as ReachDialog from '@reach/dialog'
+import { styles } from './dialog.styles'
+import { merge } from '../../utils'
+
+const DialogContent = ({ css, ...props }) => (
+  <Element
+    as={ReachDialog.DialogContent}
+    component="DialogContent"
+    css={merge(styles.DialogContent, css)}
+    {...props}
+  />
+)
+
+const DialogOverlay = ({ css, ...props }) => (
+  <Element
+    as={ReachDialog.DialogOverlay}
+    component="DialogContent"
+    css={merge(styles.DialogOverlay, css)}
+    {...props}
+  />
+)
+
+const Dialog = ({ css, children, ...props }) => (
+  <Element
+    as={ReachDialog.DialogOverlay}
+    component="DialogOverlay"
+    css={merge(styles.DialogOverlay, css)}
+    {...props}
```

Comment:
Do we want to add the same `props` to both elements? That could lead to problems if it adds duplicated props, imagine a `data-attribute` or an `id`

Reply:
Yes makes sense, I removed the `props` from the overlay and have it now only on the `DialogContent` component, since this is the "core" component, I think. Maybe it would make sense to add a `props` called `overlayProps` to add props to the overlay, what do you think?